src/Angor/Avalonia/Angor.Contexts.Funding/Investor/IInvestmentAppService.cs

```diff
@@ -8,6 +9,7 @@ namespace Angor.Contexts.Funding.Investor;
 public interface IInvestmentAppService
 {
     Task<Result<IEnumerable<InvestmentDto>>> GetInvestments(ProjectId projectId);
-    Task<Result<CreateInvestment.Draft>> CreateDraft(Guid sourceWalletId, ProjectId projectId, Amount amount);
-    Task<Result<Guid>> RequestInvestment(Guid sourceWalletId, ProjectId projectId, CreateInvestment.Draft draft);
+    Task<Result<CreateInvestment.Draft>> CreateInvestmentDraft(Guid sourceWalletId, ProjectId projectId, Amount amount);
+    Task<Result<Guid>> Invest(Guid sourceWalletId, ProjectId projectId, CreateInvestment.Draft draft);
+    Task<Result<IEnumerable<GetPendingInvestments.PendingInvestmentDto>>> GetPendingInvestments(Guid walletId, ProjectId projectId);
```

Comment:
The name should be single because the project id is only ever for 1 investment at the time